server/controllers/assess/assessmentsController.ts

```diff
@@ -35,10 +35,41 @@ export default class AssessmentsController {
       } else {
         res.render('assessments/show', {
           assessment,
-          pageHeading: 'Assess an Approved Premises (AP) application',
+          pageHeading: tasklistPageHeading,
           sections: getSections(assessment),
         })
       }
     }
   }
+
+  submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const assessment = await this.assessmentService.findAssessment(req.user.token, req.params.id)
+
+      if (req.body?.confirmation !== 'confirmed') {
```

Comment:
Good catch! I don't think we do this yet in Apply

Reply:
I'll add it in later :)